src/test/java/uk/gov/justice/hmpps/prison/repository/BookingRepositoryTest.java

```diff
@@ -126,22 +122,22 @@ public void testCreateBookingAppointmentWithEndComment() {
     }
 
     @Test
-    public void testGetBookingVisitLastSameDay() {
-        final var visit = repository.getBookingVisitLast(-1L, LocalDateTime.parse("2016-12-11T16:00"));
+    public void testGetBookingVisitNextSameDay() {
```

Comment:
there were no repository tests for next visits, so moved tests to test that instead